hbase_sql/pom.xml

```diff
@@ -148,7 +148,12 @@
             <artifactId>guava</artifactId>
             <scope>provided</scope>
         </dependency>
-
+        <dependency>
+            <groupId>org.iq80.snappy</groupId>
```

Comment:
Splice explicitly uses _org.xerial.snappy_ (in addition to Hadoop's internal version). Why is this dependency needed?

Reply:
org.iq80.snappy.Snappy is still used in hbase_sql/com.splicemachine.orc.stream.OrcInputStream. Before spark3 it was found via spark-hive_2.11:jar:2.4.0-cdh6.3.0 -> hive-exec:jar:2.1.1-cdh6.3.0.
Looks like there is only one class with this dependency. Should I replace it by org.xerial.snappy.Snappy?

Reply:
Use _com.splicemachine.compression.SpliceSnappy_ from _spliceengine/utilities_. It's based on _org.xerial.snappy_.

Reply:
Done